lucuma-schemas/src/clue/scala/lucuma/schemas/ObservationDB.scala

```diff
@@ -87,7 +87,7 @@ trait ObservationDB {
     type CallForProposalsType                = enums.CallForProposalsType
     type CatalogName                         = enums.CatalogName
     type ChargeClass                         = enums.ChargeClass
-    type CloudExtinction                     = enums.CloudExtinction
+    type CloudExtinctionPreset               = CloudExtinction.Preset
```

Comment:
in the schema do we ever only get enum instances?

Reply:
Only enums so far, yes.